packages/common/src/graphql/entities.ts

```diff
@@ -101,19 +108,15 @@ export function getAllEntitiesRelations(_schema: GraphQLSchema | string): GraphQ
         throw new Error(`${typeString} is not an valid type`);
       }
       // handle indexes
-      const indexDirectiveVal = getDirectiveValues(indexDirective, field.astNode);
       if (indexDirectiveVal) {
         if (typeString !== 'ID' && Object.values(FieldScalar).includes(typeString)) {
           newModel.indexes.push({
             unique: indexDirectiveVal.unique,
             fields: [field.name],
           });
         } else if (typeString !== 'ID' && entityNameSet.includes(typeString)) {
-          newModel.indexes.push({
-            unique: indexDirectiveVal.unique,
-            fields: [`${field.name}Id`],
-            using: IndexType.HASH,
-          });
+          //is foreign key
+          continue;
```

Comment:
`continue` seems too dangerous be used here. You only want to skip this if block but `continue` skip the while block

Reply:
you can do nothing here, or just print a warning saying `@index` here is ignored